packages/lib-classifier/src/components/Classifier/components/TaskArea/components/DisabledTaskPopup/DisabledTaskPopup.js

```diff
@@ -61,6 +61,8 @@ export default function DisabledTaskPopup({
   return (
     <StyledDialog open={active} aria-label={t('TaskArea.DisabledTaskPopup.title')}>
       <Modal
+        role={undefined}
+        aria-label={undefined}
```

Comment:
`Modal` here is being used to render content for a `<dialog>` element, so we have to unset its default dialog role and accName.

Reply:
It might be better to fix this in the `withLayer` helper, so that component authors are free to use the undecorated `Modal` component inside a native `<dialog>` if they wish.